bot/code_review_bot/report/phabricator.py

```diff
@@ -226,11 +226,16 @@ def publish(self, issues, revision, task_failures, notices, reviewers):
             and not notices
         ):
             # Nothing changed, no issue have been opened or closed
-            logger.warning(
-                "No new issues nor failures/notices were detected. "
-                "Skipping comment publication (some issues are unresolved)",
-                unresolved_count=len(unresolved_issues),
-            )
+            if len(unresolved_issues) == 0:
+                logger.info(
+                    "No new issues nor failures/notices were detected. Skipping comment publication."
+                )
+            else:
+                logger.warning(
```

Comment:
@La0 Shouldn't we always use `logger.info` ?
It seems like an expected behavior to me (not commenting in Phabricator when issues are still unresolved).

Reply:
You're right, you can downgrade to info in all cases